python/graphstorm/model/gnn_encoder_base.py

```diff
@@ -158,24 +158,46 @@ def dist_minibatch_inference(g, gnn_encoder, get_input_embeds, batch_size, fanou
                                                             shuffle=False,
                                                             drop_last=False)
 
-        for iter_l, (input_nodes, output_nodes, blocks) in enumerate(dataloader):
+        len_dataloader = max_num_batch = len(list(dataloader))
```

Comment:
In edge_gnn we use len_loader, here we use len_dataloader. Can we unify the naming?

Reply:
Sure